VOSpace.tex

```diff
@@ -515,6 +515,7 @@ \subsubsection{Standard properties}
     \item ivo://ivoa.net/vospace/core\#mtime SHALL be used as the property URI denoting the data modification time
     \item ivo://ivoa.net/vospace/core\#ctime SHALL be used as the property URI denoting status change (aka metadata modification) time
     \item ivo://ivoa.net/vospace/core\#btime SHALL be used as the property URI denoting initial creation time
+    \item ivo://ivoa.net/vospace/core\#immutable MAY be used as the property URI denoting a node whose content cannot be changed and for which moveNode and deleteNode operations are not allowed. If the node is a container node, child nodes cannot be added or removed. Children of an immutable node inherit this property. Properties that are not readOnly can still be edited
```

Comment:
Move the details of what `immutable` means to a separate section, giving us more room to describe it in detail.
```suggestion
    \item ivo://ivoa.net/vospace/core\#immutable MAY be used as the property URI denoting a node whose content cannot be changed and for which moveNode and deleteNode operations are not allowed. See section \ref{tbd} for details.
```

Reply:
I created a new section. I put it under "Nodes and node types" section, since it describes a special nodes behavior. Let me know if you prefer to move it somewhere else, for example at the end of the properties section. I added also a condition for link nodes (target can't be changed), but I'm not sure if it should be possible to change the target using the setNode operation.

Reply:
Looks good. :+1:

Reply:
> I'm not sure if it should be possible to change the target using the setNode operation.

The target is effectively the content for a LinkNode, so it once the node is made immutable it you shouldn't be able to direct the link to point to something else.